src/bench/bench.cpp

```diff
@@ -88,7 +88,7 @@ bool benchmark::State::KeepRunning()
     lastCycles = nowCycles;
     ++count;
 
-    if (now - beginTime < maxElapsed) return true; // Keep going
+    if (now - beginTime < maxElapsed || count == 1) return true; // Keep going
```

Comment:
If `count` would be guaranteed to be different from zero, then the check on line 50 (`if (count == 0) {`) wouldn't be needed, right? :-)

If the branch `if (count == 0) {` is taken then the relevant parts of the remaining code can be reduced to something like:

```
count = 0; // implicit
++count;
--count;
double average = (now-beginTime)/count;
```

Let me know if I'm missing something obvious, this is one of my first contributions so please bear with me :-)